cookbooks/bcpc-hadoop/recipes/configs.rb

```diff
@@ -62,9 +64,69 @@
 # Install Java
 include_recipe "bcpc-hadoop::java_config"
 include_recipe "java::default"
+include_recipe "java::oracle_jce"
 
 %w{zookeeper}.each do |pkg|
   package pkg do
     action :upgrade
   end
 end
+
+# Create Keytabs (if kerberos is eanbled)
+if node[:bcpc][:hadoop][:enable_kerberos] == true then
```

Comment:
Moved to `node[:bcpc][:hadoop][:kerberos]` and changed the attribute to be `enable`.